shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/resultset/ShardingSphereResultSet.java

```diff
@@ -407,7 +408,7 @@ public <T> T getObject(final int columnIndex, final Class<T> type) throws SQLExc
             return (T) getBlob(columnIndex);
         } else if (Clob.class.equals(type)) {
             return (T) getClob(columnIndex);
-        } else if (LocalDateTime.class.equals(type) || LocalDate.class.equals(type) || LocalTime.class.equals(type)) {
```

Comment:
The judgment condition of this piece is too complicated. I think it is better to judge only `java.time.temporal.TemporalAccessor`.